README.md

```diff
@@ -161,6 +161,18 @@ JSON.stringify(img._corners)
 => "[{"lat":51.52,"lng":-0.1},{"lat":51.52,"lng":-0.14},{"lat":51.5,"lng":-0.1},{"lat":51.5,"lng":-0.14}]"
 ```
 
+### Adding "DOMStrings" for custom toolbars
+
+Custom "DOMStrings" **(i.e., strings of HTML)** that define the custom toolbars can easily be added by describing those in the `src/edit/tools/DistortableImage.Guides.js` file and referencing those in the `addToolbar` method inside the `src/edit/DistortableImage.Edit.js` file.
+
+```js
+var guide_strings = [dom_string]; // add guides here for custom toolbars
+
+L.DistortableImage.Guides = guide_strings;
+```
+
+**Note:** The default `toolbarStyle` will initially be set to "Popup", unless specified otherwise.
```

Comment:
```suggestion
**Note:** The default `toolbarStyle` will initially be set to "Popup", unless specified otherwise when initializing the image. 
```

Then show an example of this!